plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -448,11 +445,7 @@ public static function get_update_data() {
 		$payload = array();
 
 		// Scan subscriptions.
-		try {
-			$subscriptions = WC_Helper::get_subscriptions();
-		} catch ( Exception $e ) {
-			$subscriptions = array();
-		}
+		$subscriptions = WC_Helper::get_subscriptions();
```

Comment:
Not handling the exception here can probably cause errors in many places, as `get_subscriptions` can throw an exception whenever there is a non 200 response.

Reply:
> I had to keep @throws in the docblock because phpcs was complaining.

Can we rather then ignore the phpcs error with a comment? Keeping `@throws` declaration in the docblock is quite strong indication that there can be errors

Reply:
Good point. 85f7641350a011b90181ced2ea076b3fc3936b62